### ISSUE TITLE
Using Glyph Should Display Who Uses It

### ISSUE THREAD

**Issue by billmcdulin** [ValveSoftware/Dota2-Gameplay#12759]
### Description

Using Glyph Should Display Who Uses It

**Comment by TobyFlendersonFromHR** [comment]
yeah, that way we can blame our teammates more and feel better about ourselves

**Comment by Avarise831** [comment]
#581 #1193

**Comment by billmcdulin** [comment]
It is more for awareness than anything. Especially comes into play for example in ranked games. Some people just glyph because they are griefing which IMO should lead to a report.

**Comment by TobyFlendersonFromHR** [comment]
yeah so more blame to throw around

**Comment by billmcdulin** [comment]
typical Toby comment vs helping the players playing the game
